packages/backend/src/storage/storage.ts

```diff
@@ -294,17 +296,29 @@ export class Storage {
     if (repo && !repo.eventsAttached) {
       log('Subscribing to channel ', channelData.address)
 
-      db.events.on('write', (_address, entry) => {
+      db.events.on('write', async (_address, entry) => {
+        const message = entry.payload.value
+        const crypto = getCrypto()
+        const signature = stringToArrayBuffer(message.signature)
+        const cryptoKey = await keyObjectFromString(message.pubKey, crypto)
+        const verify = await verifySignature(signature, message.message, cryptoKey)
         log(`Writing to public channel db ${channelData.address}`)
         this.io.loadMessages({
-          messages: [entry.payload.value]
+          messages: [entry.payload.value],
+          verifyStatus: verify
```

Comment:
I think "verified" would be a bit better since it's just a boolean. verifyStatus suggests that there are multiple statuses.